experiments/AtmosLES/stable_bl_model.jl

```diff
@@ -329,7 +332,7 @@ function stable_bl_model(
         problem = problem,
         turbulence = SmagorinskyLilly{FT}(C_smag),
         moisture = moisture,
-        source = source_default,
+        source = source,
```

Comment:
I think this is source_default in the convective_bl case, in case we want to homogenize notation

Reply:
No, this needs to be `source`, otherwise it'll break the `NonEquil` moisture model case (look above).